code/datums/stars.dm

```diff
@@ -101,6 +101,11 @@
 			no_unload_reason = "RELAY"
 			return 0
 
+	for(var/obj/machinery/computer/shuttle/fob/F in GLOB.machines)
+		if((F.z in z_levels) && F.z != ZLEVEL_STATION)
+			no_unload_reason = "FOB SHUTTLE"
+			return 0
```

Comment:
no, you remove this code entirely from here and set the unload reason when the fob lands and unset and unload when it leaves.